components/controls/buttons_usb_encoder/setup-buttons-usb-encoder.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+HOME_DIR="/home/pi"
+JUKEBOX_HOME_DIR="${HOME_DIR}/RPi-Jukebox-RFID"
+BUTTONS_USB_ENCODER_DIR="${JUKEBOX_HOME_DIR}/components/controls/buttons_usb_encoder"
+
+question() {
+    local question=$1
+    read -p "${question} (y/n)? " choice
+    case "$choice" in
+      y|Y ) ;;
+      n|N ) exit 0;;
+      * ) echo "Error: invalid" ; question ${question};;
+    esac
+}
+
+printf "Please make sure that the Buttons USB Encoder and the buttons are connected before continuing...\n"
+question "Continue"
+
```

Comment:
Are all Python dependencies available out of the box or do we need a requirements.txt file with all needed packages and a command here to install these from requirements.txt?

Reply:
there is no need of additional dependencies. everything works out of the box :)